static/Makefile

```diff
@@ -0,0 +1,29 @@
+# List any generated files here
+TARGETS = static_gen.go
+# List any source files used to generate the targets here
+SOURCES = data/build data/swagger.json
+
+# Default target
+all: $(TARGETS) data/build data/swagger.json
+
+# Clean all targets recursively
+clean:
+	rm -f $(TARGETS)
+	rm -r data
+
+# Target for the built UI assets directory.
+data/build:
+	../scripts/fetch-ui-assets.sh
+
+# Target for the swagger.json from the openapi repo
+data/swagger.json:
+	OPENAPI_SHA=c87a08f832e79bc338a37509028641cda9e1875b ../scripts/fetch-swagger.sh
```

Comment:
I don't feel super great about how we are going to keep track of the specific magic strings that link these things together. This one is for the commit in the `openapi` repo that we want to pull the swagger from. There's not currently one for the master branch for the UI version since it always just pulls the `master` release tag, but there will be one for release branches.

My thought was that we could make the fetcher scripts take arguments (env vars), and then give them the env vars _somehow_, which would be specified in a central location. My first guess was within this `static` package `Makefile` but there's probably a better way.

Reply:
I'd typically `declare -r OPENAPI_SHA` in the `fetch-swagger.sh` script instead of tracking it here. That'd be ~consistent with how `fetch-ui-assets.sh` works.